### ISSUE TITLE
Double file loading

### ISSUE THREAD

**Issue by xeqi** [bhauman/lein-figwheel#108]
## Reproduction
```
$ lein new figwheel ex
$ cd ex
$ sed -i 's/-SNAPSHOT//' project.clj   #Use figwheel 0.2.5
$ lein figwheel
Wait for compile and connect browser
$ echo " " >> src/ex/core.cljs 
```

## Output

Chrome Developer Console:
```
 Edits to this text should show up in your developer console.
utils.cljs?rel=1426529316470:30 Figwheel: trying to open cljs reload socket
utils.cljs?rel=1426529316470:30 Figwheel: socket connection established
utils.cljs?rel=1426529316470:30 Figwheel: notified of file changes
core.cljs?rel=1426529314793:57 Edits to this text should show up in your developer console.
core.cljs?rel=1426529314793:57 Edits to this text should show up in your developer console.
utils.cljs?rel=1426529316470:30 Figwheel: loaded these files
utils.cljs?rel=1426529316470:32 ("ex/core.js" "ex/core.js")
```

Note that `Edits to this text should show up in your developer console.` happens twice, and `ex/core.js` appears twice in the loaded files list.

## Notes

The changlog mentions `^:figwheel-always was causing a double file reload when editing the marked ns`, but it appears to still happen?

**Comment by bhauman** [comment]
just fixed this in master

**Comment by JulianLeviston** [comment]
Bit of a noob question - how can we get the fix into our codebase(s)? I think I'm seeing this issue, too... along with some other really annoying warning-based half-compilation stuff which I'm not sure the origin of (using Reagent).

**Comment by idiotslowdown** [comment]
@JulianLeviston 
  1. go into ~/m2, 
  2. delete /lein-figwheel, /figwheel, /figwheel-sidecar
  3. clone repo 
  4. lein install inside repo

**Comment by bhauman** [comment]
I have just deployed this fix in 0.2.6
